lib/livebook/deployment_groups.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Livebook.DeploymentGroups do
+  # Shared deployment group functionality across all hubs.
```

Comment:
This will be a feature only for Livebook Teams, so we can remove this functionality from the root and hide inside Livebook Teams. I think this is good because we will have fewer layers of indirection.